resources/qml/Preferences/Materials/MaterialsView.qml

```diff
@@ -517,97 +515,120 @@ Item
             }
         }
 
-        Column
+        ScrollView
```

Comment:
Why not make this a `ListView` and set the model property directly on this component